modules/lvms-scaling-storage-of-clusters-using-cli.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-cli_{context}"]
+= Scaling up the storage of clusters by using the CLI
+
+You can scale up the storage capacity of the worker nodes on a cluster by using the `OpenShift` CLI.
+
+.Prerequisites
+
+* You have additional unused devices on each cluster to be used by {lvms}.
+* You have installed the OpenShift CLI (`oc`).
+* You have created an `LVMCluster` CR.
+
+.Procedure
+
+. Edit the `LVMCluster` custom resource (CR) YAML file by running the following command:
```

Comment:
```suggestion
. Edit the `LVMCluster` custom resource (CR) YAML by running the following command:
```
Here we are modifying the live object instead of the file.

Reply:
Updated as per the suggestion.